include/json/writer.h

```diff
@@ -73,49 +73,49 @@ String JSON_API writeString(StreamWriter::Factory const& factory,
 
 /** \brief Build a StreamWriter implementation.
 
-Usage:
-\code
-  using namespace Json;
-  Value value = ...;
-  StreamWriterBuilder builder;
-  builder["commentStyle"] = "None";
-  builder["indentation"] = "   ";  // or whatever you like
-  std::unique_ptr<Json::StreamWriter> writer(
-      builder.newStreamWriter());
-  writer->write(value, &std::cout);
-  std::cout << std::endl;  // add lf and flush
-\endcode
+* Usage:
+*  \code
```

Comment:
indent the \code to \endcode 1 more space to be consistent with reader.h

Reply:
you mean making \code and \encode are one more space ?

Reply:
I mean yes. The other usage or example clauses have 2-space indents for the \code and following associated block, and here there's only a 1-space indent.